app/views/shared/_tab_row.html.erb

```diff
@@ -1,9 +1,11 @@
 <% anchor = 'tabs' %>
 <div class="row" id="<%= anchor %>">
-  <% (item.tabs.keys).reject { |x| skip.include?(x) }.each do |tab| %>
-    <% is_current = tab == params[:tab] %>
-    <div class="tab col-md-2 <%= 'tab-current' if is_current %>">
-      <%= link_to_unless(is_current, tab.titlecase, "#{tab}##{anchor}") %>
-    </div>
-  <% end %>
+  <% (item.tabs.keys).reject { |x| skip.include?(x) }.tap do |tabs|
```

Comment:
@afred : It's a local supplied by the caller:
```
  <%= render partial: 'shared/tab_row', locals: {item: @item, skip: ['intro', 'author', 'extra']}%>
```
I would like, eventually, for only actual tabs to be returns by the tabs method, but it hasn't seemed urgent up till now, so this is where the h2s are distinguished.